design/async-commit/README.md

```diff
@@ -154,15 +154,15 @@ TODO
 
 ## Possible optimisations
 
-There are restrictions on the size of the transaction. For example, if the key involved in the transaction is less than 64, parallel commit is used, or a hierarchical structure is adopted. The primary lock records a few secondary locks, and these secondary locks record other secondary locks respectively. It is easy to implement, just recursion, and the cost of failure recovery needs to be considered.
+There are restrictions on the size of the transaction. For example, if the key involved in the transaction is less than 64, async commit is used, or a hierarchical structure is adopted. The primary lock records a few secondary locks, and these secondary locks record other secondary locks respectively. It is easy to implement, just recursion, and the cost of failure recovery needs to be considered.
 
 Crdb mentioned two ways to reduce the impact of recovery, and TiDB has also implemented: one is to perform commit cleanup as soon as possible when committing; the second is transaction heartbeat to prevent cleanup of alive transactions.
 
 ## Related Work
 
 ### Cockroach DB
 
-In CRDB, parallel commit extends pipelined pessimistic locking.
+In CRDB, async commit extends pipelined pessimistic locking.
```

Comment:
CockroachDB uses the wording "parallel commit".

Reply:
👌